src/has.ts

```diff
@@ -64,6 +64,7 @@ add('dom-mutationobserver', function(): boolean {
 add('microtasks', function () {
 	return has('promise') || has('host-node') || has('dom-mutationobserver');
 });
+add('object-assign', typeof (<any> Object).assign === 'function');
```

Comment:
I vaguely remember that discussion, but also vaguely remember no definitive consensus, so I decided to follow the convention already in place.